usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/WeightManagerClient.java

```diff
@@ -0,0 +1,201 @@
+package org.pesmypetcare.usermanagerlib.clients;
+
+import com.google.gson.Gson;
+
+import org.json.JSONObject;
+import org.pesmypetcare.usermanagerlib.datacontainers.DateTime;
+import org.pesmypetcare.usermanagerlib.datacontainers.Weight;
+import org.pesmypetcare.usermanagerlib.datacontainers.WeightData;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+public class WeightManagerClient {
+    public static final String WEIGHTNAME = "weightName";
+    public static final String WEIGHT = "weight";
+    private static final String BASE_URL = "https://pes-my-pet-care.herokuapp.com/weight/";
+    private static final String POST = "POST";
+    private static final String GET = "GET";
+    private static final String DELETE = "DELETE";
+    private static final String PUT = "PUT";
+    private static Gson GSON = new Gson();
+    private TaskManager taskManager;
+
+    public WeightManagerClient() {
+        taskManager = new TaskManager();
+    }
+
+
+    /**
+     * Creates a weight of a pet on the database.
+     * @param accessToken The personal access token for the account
+     * @param owner Username of the owner of the pet
+     * @param petName Name of the pet
+     * @param weight The weight entity that contains the attributes of the weight
+     * @param date The date of creation
+     * @return The response code
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public int createWeight(String accessToken, String owner, String petName, Weight weight,
+                            DateTime date) throws ExecutionException, InterruptedException {
+        JSONObject reqJson = weight.getBody().buildMealJson();
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(POST);
+        taskManager.setReqBody(reqJson);
+        StringBuilder response = taskManager.execute(BASE_URL + owner + "/" + petName + "/" + date,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/" appears 12 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5361615)